ocfweb/docs/docs/staff/procedures/toner-handling.md

```diff
@@ -0,0 +1,11 @@
+[[!meta title="Recycling Used Toner"]]
+
+OCF goes through toners quite fast, due to the heavy volume of printing from our users. In the past, we've let it accumulate inside the storage room which resulted in accidental toner spillage. The following document contains how to recycle the toners and hope to encourage rapid removal of the toners from OCF space.
+
+## Options to dispose the toners
+
+* We can dispose the toners by returning them to [Staples](http://rewards.staples.com/SRC/faqs.do) down at the Shattuck Ave. They give us $2 Staples rewards per toner, and there is maximum limit of 10 toners per month.
```

Comment:
Do we need to supply our Staples rewards account? Should give info about where to find that information.

Reply:
Also, should mention things we can buy staples rewards with (snax) to make it more incentived

Reply:
ty, that's a good info to document